apps/examples/testcase/le_tc/kernel/tc_task.c

```diff
@@ -238,9 +293,24 @@ static void tc_task_task_create(void)
 	int pid;
 	const char *task_param[2] = { TEST_STRING, NULL };
 	g_callback = ERROR;
+
+	/* Inavlid priority value check */
+
+	pid = task_create("tc_task_create", SCHED_PRIORITY_MIN - 1, 1024, create_task, (char * const *)task_param);
+	TC_ASSERT_EQ("task_create", pid, ERROR);
+	TC_ASSERT_EQ("task_create", errno, EINVAL);
+
+	/* Regular functionality check */
+
 	pid = task_create("tc_task_create", SCHED_PRIORITY_MAX - 1, 1024, create_task, (char * const *)task_param);
 	TC_ASSERT_GT("task_create", pid, 0);
 	TC_ASSERT_EQ("task_create", g_callback, OK);
+
+	/* New task creation failure check due to maximum task creation limit (>CONFIG_MAX_TASKS)*/
+	pid = task_create("tc_maxtask", SCHED_PRIORITY_MAX - 1, 1024, max_task, (char * const *)NULL);
```

Comment:
You make a task to test making maximun task. But that's all. There is no checking result logic (no TC_ASSERT macro).